src/flush-microtasks.js

```diff
@@ -58,6 +58,17 @@ try {
           'if you encounter this warning.',
       )
     }
+
+    if (isModernScheduleCallbackSupported) {
+      // eslint-disable-next-line no-console
+      console.error(
+        'This environment does not support module requiring, so we cannot require the Scheduler. ' +
+          'To fix this you can require the Scheduler in your test setup file.',
+      )
+
+      // Fallback to no scheduleCallback Supported.
+      isModernScheduleCallbackSupported = false
+    }
   }
 }
```

Comment:
It's not whether we can require the scheduler, but that there is no scheduler available. So I think we can make this change and that'll help resolve all the issues.

```suggestion

  }
}

const isModernScheduleCallbackSupported = Scheduler && satisfies(React.version, '>16.8.6', {
  includePrerelease: true,
})
```

Reply:
What do you mean when you say available? Since they use React 16.13.1, it should be available shouldn't it?
To me it looks like we couldn't require it using `nodeRequire.call(module, 'scheduler')`.